src/main/java/com/zaxxer/nuprocess/windows/ProcessCompletions.java

```diff
@@ -122,7 +125,8 @@ public void run()
       }
       catch (Exception e) {
          // TODO: how to handle this error?
-         e.printStackTrace();
```

Comment:
This is the last `e.printStackTrace` in the production NuProcess code; the last two are in tests (which seems fine)